src/app/shared/components/policy-link/policy-link.component.html

```diff
@@ -0,0 +1,8 @@
+<a
+  [href]="'https://pomagamukrainie.gov.pl/assets/rodo/politykaprywatnosci' + getSuffixForTranslatedResource() + '.pdf'"
+  target="_blank"
+  class="navbar-item"
+>
+  <ng-template [ngIf]="denominator">{{ "POLICY" | translate }}</ng-template>
+  <ng-template [ngIf]="!denominator">{{ "PRIVATE_POLICY" | translate }}</ng-template>
```

Comment:
How about:
```suggestion
  <ng-template>{{ (denominator ? "POLICY" : "PRIVATE_POLICY") | translate }}</ng-template>
```

Reply:
Right, thanks! updated in d4fd02f2, even shorter.